src/plugins/displayLayout/components/SubobjectView.vue

```diff
@@ -142,7 +142,7 @@ export default {
             this.removeSelectable();
         }
 
-        if (this.domainObject.isMutable) {
+        if (this.domainObject && this.domainObject.isMutable) {
```

Comment:
Good catch! this didn't occur to me when I wrote this code. 

There's a memory leak here though. Because we're not cancelling the fetch, the promise may eventually resolve with a `MutableDomainObject` that will never be destroyed, it will just linger around until the end of time (or the user refreshes the browser).

I think what we need to do is store the promise returned by `getMutable`, and in `beforeDestroy` chain a `then` handler that will call `destroyMutable` (or do nothing if no object is ultimately returned).